### ISSUE TITLE
Error when trying to create a record_numericalizer in the Deduplication notebook

### ISSUE THREAD

**Issue by azfarulis** [vintasoftware/entity-embed#30]
Hi vintasoftware,

This is my first time writing up a bug report.

I was trying out the Deduplication notebook on Google Colab (GPU + High-ram shape), and when I get to the stage for creating **record_numericalizer** it throws an `AttributeError` saying that:

```python
'Vocab' object has no attribute 'load_vectors'
```

Here is a screenshot attached:

![Screenshot 2021-07-09 at 23 12 47](https://user-images.githubusercontent.com/71644301/125137373-d6879900-e10c-11eb-95fe-dd0a8fca131b.png)

I was following the notebook as structured without any deviation.

Please let me know if I can add any more information to help.

**Comment by fjsj** [comment]
Thanks for the report, I'll check this in the next couple of days.

**Comment by fjsj** [comment]
@azfarulis there has been a change in torchtext that broke it's API. I'll be fixing the requirements today. Thanks for reporting the issue. A quick solution right now is forcing the right versions in your Colab by adding below the "Boilerplate" section a new cell including this:

```
!pip install "torch>=1.7.1,<1.9"
!pip install "torchtext>=0.8.1,<0.9"
```

**Comment by azfarulis** [comment]
@fjsj Thanks for the heads up. Will try it out and report back!

**Comment by fjsj** [comment]
@azfarulis 
This issue is now fixed on main branch and released as version 0.0.6

Please factory reset your runtimes at Colab and try again. Closing this issue for now. Thanks again for reporting the issue! Please let us know if you find other problems.
